curriculum.md

```diff
@@ -39,7 +39,41 @@ Here are some basic programming videos that are pretty good, but are done in Pyt
 
 ---
 
-## Week 3: React applications!
+## Week 3: More JavaScript and the DOM
+### Learning objective
+- Complementary JS skills for working with the DOM: 
+   - functions
+   - objects and arrays
+   - events
+   - math functions
+   - this/context
+   - jquery
```

Comment:
Do we want to go to jQuery? Feels a bit outdated. 🤔

Reply:
Fixed!